src/utilities/status.ts

```diff
@@ -3,12 +3,14 @@ export enum Status {
   Complete = 'Complete',
   Failed = 'Failed',
   Incomplete = 'Incomplete',
+  Unchecked = 'Unchecked',
   Modified = 'Modified',
   Pending = 'Pending',
-  PartialSuccess = 'PartialSuccess',
+  PartialSuccess = 'Partial Success',
```

Comment:
Not super relevant and more of a cleanup question: we have an `/enums` directory, should this enum go under there?

Reply:
Good idea, will move it.